crates/jstz_cli/src/sandbox/mod.rs

```diff
@@ -31,9 +31,9 @@ pub enum Command {
 }
 
 pub async fn start(detach: bool, background: bool) -> Result<()> {
-    let mut cfg = Config::load()?;
+    let cfg: &'static mut Config = Box::leak(Box::new(Config::load()?));
```

Comment:
Definitely a code smell that we're leaking memory 😅

Reply:
😆